wix/alacritty.wxs

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" 
+    xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+
+    <Product Name="Alacritty" Id="*" UpgradeCode="87c21c74-dbd5-4584-89d5-46d9cd0c40a7" Language="1033" Codepage="1252" Version="0.2.9" Manufacturer="Alacritty">
+        <Package InstallerVersion="200" Compressed="yes" InstallScope="perMachine"/>
+        <MajorUpgrade AllowSameVersionUpgrades="yes" DowngradeErrorMessage="A newer version of [ProductName] is already installed."/>
+        <Icon Id="AlacrittyIco" SourceFile="..\assets\windows\alacritty.ico"/>
+        <WixVariable Id="WixUILicenseRtf" Value="license.rtf"/>
+        <Property Id="ARPPRODUCTICON" Value="AlacrittyIco"/>
+        <MediaTemplate EmbedCab="yes"/>
+        <UIRef Id="WixUI_Minimal"/>
+
+        <Feature Id="ProductFeature" Title="ConsoleApp" Level="1">
+            <ComponentRef Id="AlacrittyExe"/>
+            <ComponentRef Id="WinPtyAgentExe"/>
+            <ComponentRef Id="AlacrittyConfig"/>
+            <ComponentRef Id="AlacrittyShortcut"/>
+            <ComponentRef Id="ModifyPathEnv"/>
+            <ComponentRef Id="ContextMenu"/>
+        </Feature>
+
+        <!-- Create directories -->
+        <Directory Id="TARGETDIR" Name="SourceDir">
+            <Directory Id="ProgramFiles64Folder">
+                <Directory Id="AlacrittyProgramFiles" Name="Alacritty"/>
+            </Directory>
+            <Directory Id="AppDataFolder">
+                <Directory Id="AlacrittyAppData" Name="alacritty"/>
+            </Directory>
+            <Directory Id="ProgramMenuFolder">
+                <Directory Id="AlacrittyProgramMenu" Name="Alacritty"/>
+            </Directory>
+        </Directory>
+
+        <!-- Application binaries -->
+        <DirectoryRef Id="AlacrittyProgramFiles">
+            <Component Id="AlacrittyExe" Guid="*">
+                <File Id="AlacrittyExeFile" Source="..\target\release\alacritty.exe" Name="alacritty.exe" KeyPath="yes"/>
+            </Component>
+            <Component Id="WinPtyAgentExe" Guid="*">
+                <File Id="WinPtyAgentExeFile" Source="..\target\release\winpty-agent.exe" Name="winpty-agent.exe" KeyPath="yes"/>
+            </Component>
+        </DirectoryRef>
+
+        <DirectoryRef Id="AlacrittyAppData">
+            <!-- Copy default config -->
+            <Component Id="AlacrittyConfig" Guid="*" Permanent="yes">
+                <File Id="AlacrittyConfigFile" Source="..\alacritty.yml" Name="alacritty.yml" KeyPath="yes"/>
+            </Component>
+        </DirectoryRef>
```

Comment:
We don't actually want to copy over the configuration file. It's generated automatically at the right location once Alacritty is started, there's no point to duplicate this here.

Reply:
True, okay I will remove this part.